From 5d454e56745e7a210abca17649a815eec7009e0f Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:32 -0400
Subject: [cpufreq] Remove extra blank line

Message-id: <1409235879-1283-59-git-send-email-prarit@redhat.com>
Patchwork-id: 89525
O-Subject: [RHEL7.1 PATCH BZ 1134639 058/185] cpufreq: Remove extra blank line
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit bbdd04ab1f375ef46a0e2d98de439863d35e4d3e
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Wed Oct 2 14:13:12 2013 +0530

    cpufreq: Remove extra blank line

    We don't need a blank line just at start of a block, lets remove it.

    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Reviewed-by: Srivatsa S. Bhat <srivatsa.bhat@linux.vnet.ibm.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index aa83a3f..8c2a542 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -1199,7 +1199,6 @@ static int __cpufreq_remove_dev_prepare(struct device *dev,
   if (!frozen)
    sysfs_remove_link(&dev->kobj, "cpufreq");
  } else if (cpus > 1) {
-
   new_cpu = cpufreq_nominate_new_policy_cpu(policy, cpu, frozen);
   if (new_cpu >= 0) {
    update_policy_cpu(policy, new_cpu);
-- 
1.7.1